locale/es/docs/es6.md

```diff
@@ -1,45 +1,45 @@
 ---
-title: ECMAScript 2015 (ES6) and beyond
+title: ECMAScript 2015 (ES6) y más allá
 layout: docs.hbs
 ---
 
-# ECMAScript 2015 (ES6) and beyond
+# ECMAScript 2015 (ES6) y más allá
 
-Node.js is built against modern versions of [V8](https://v8.dev/). By keeping up-to-date with the latest releases of this engine, we ensure new features from the [JavaScript ECMA-262 specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm) are brought to Node.js developers in a timely manner, as well as continued performance and stability improvements.
+Node.js está construido en base a versiones modernas de [V8](https://v8.dev/). Al mantenernos actualizados con las últimas versiones de este motor, aseguramos nuevas características del [La Especificación ECMA-262 de JavaScript](http://www.ecma-international.org/publications/standards/Ecma-262.htm) que proporciona a los desarrolladores de Node.js, así como mejoras continuas en el rendimiento y la estabilidad.
 
-All ECMAScript 2015 (ES6) features are split into three groups for **shipping**, **staged**, and **in progress** features:
+Todas las características de ECMAScript 2015 (ES6) se dividen en tres grupos <strong x-id = "1">shipping</strong>, <strong x-id = "1">staged</strong> y <strong x- id = "1">in progress</strong>:
```

Comment:
These `<strong x-id = "1">` all need to be replaced back to the original formatting style of `**words**

Reply:
Crowdin team already looking on that, please wait for it;)